src/test/java/org/grizz/service/PlanetServiceTest.java

```diff
@@ -0,0 +1,65 @@
+package org.grizz.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.grizz.OgameCloneApplication;
+import org.grizz.config.security.SecurityConfig;
+import org.grizz.exception.MaxNumberOfPlanetException;
+import org.grizz.model.Planet;
+import org.grizz.model.User;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.junit.Assert.assertThat;
+
+@Slf4j
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = {OgameCloneApplication.class, SecurityConfig.class})
+public class PlanetServiceTest {
+
+
+    @Autowired
+    @InjectMocks
```

Comment:
You inject mocks once into tested class and annotate with `Mock` all it's dependencies. It hase no sense here